packages/datadog-instrumentations/src/graphql.js

```diff
@@ -175,7 +175,7 @@ function wrapExecute (execute) {
           docSource: documentSources.get(document)
         })
 
-        const context = { source, asyncResource, fields: {} }
+        const context = { source, asyncResource, fields: {}, resolvers: {} }
```

Comment:
Why are the resolvers stored instead of being verified right away? This introduces memory overhead by requiring to keep the memory around until the entire operation is finished.

Reply:
Well, since there were already an instrumentation for inspecting the resolvers I considered that it'd be better to take advatage of that at the expense of taking a little bit of memory for the sake of response time.
Do you consider that it'd be better to traverse the GraphQL document during the parsing phase even if it cost us a few more cycles?